scripts/syncStrats.ts

```diff
@@ -169,14 +167,15 @@ async function main() : Promise< void> {
     return;
   }
 
-  //(TODO, convert to a map-like object for efficient downstream lookups and removal 
-  // handling)
+  //(TODO, convert to a map-like object for efficient downstream lookups and 
+  //  removal handling)
   try {
-    stratsToHarvest = <IStratToHrvst[]> require( '../data/stratsToHrvst.json');
+    stratsToHarvest = <IStratToHrvst[]> require( 
+                                                '../data/stratsToHarvest.json');
   } catch (error: unknown)  {
     if (!( (( testError: unknown): testError is NodeJS.ErrnoException => 
```

Comment:
I've never seen a more complicated way to catch for something and also after you get to understand what you are after you deny the expression, it's an obscure way to write it

Reply:
Again, the type-guard pattern. It's not obscure. It's TypeScript's means of dynamically narrowing the typing of a token.